src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/ViewDraftEvidenceNote.cshtml

```diff
@@ -1,13 +1,13 @@
 ﻿
 @using EA.Weee.Core.AatfEvidence
 @using EA.Weee.Web.Areas.Aatf.Controllers
-@using EA.Weee.Web.Infrastructure
 @using EA.Weee.Web.RazorHelpers
 @using MvcNavigationHelpers
+@using EA.Weee.Web.Areas.Aatf.ViewModels;
 @model EA.Weee.Web.Areas.Aatf.ViewModels.ViewEvidenceNoteViewModel
 @{
     var title = Model.Status == NoteStatus.Draft ? "Draft evidence note" : "Submitted evidence note";
-
+    var navigateBackToSelectedTab = Model.Status == NoteStatus.Draft ? ManageEvidenceOverviewDisplayOption.EditDraftAndReturnedNotes : ManageEvidenceOverviewDisplayOption.ViewAllOtherEvidenceNotes;
```

Comment:
I think the only case we might need to check (ask Sarah( is when the evidence summary tab is in place. The user is on the evidence summary tab, the user creates a new evidence note. They click the the "back" button should it take the user to the evidence summary tab or the edit/view tab?

Reply:
I'll have a chat with Sarah.